### DISCUSSION TITLE
[Bug] [ignore me, irrelevant]

### DISCUSSION THREAD

**Discussion by 26lake** [ScoopInstaller/Scoop#5646] [Help and Questions]
Back after circumventing securly
#### Current Behavior
https://pastebin.com/FGpqJ2Sa

#### Expected Behavior
For scoop to install as usual.

### System details

**Windows version:** [e.g. 7, 8, 10, 11]
10

**OS architecture:** [e.g. 32bit, 64bit, arm64]
64bit

**PowerShell version:** [output of `"$($PSVersionTable.PSVersion)"`]
5.1.19041.3031

**Comment by rashil2000** [comment]
What's C:\Users\16246\killme.ps1?

**Reply by 26lake** [reply]
it was a way i was getting past a proxy, yeah sorry for the +9m late response. ngl keep in mind this was my school laptop i was trying to get neofetch on for a cool lil coding setup but eventually i gave up.
